Xplat/src/main/resources/assets/packages/lang/zh_cn.json

```diff
@@ -0,0 +1,62 @@
+{
+	"itemGroup.packages.group": "包裹",
+
+	"block.packages.package_maker": "打包台",
+	"block.packages.package": "package",
```

Comment:
Does this intentionally start with a lowercase letter? It looks a bit weird to me, but i don't know the conventions in the Chinese minecraft communnity

Reply:
Oh, it's my mistake, thanks for the reminder, I uploaded the wrong version, this is my first translation and I will submit the correct version.